TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Mqs/EditMq/Status/EditMqResultState.cs

```diff
@@ -22,7 +22,7 @@ public class EditMqResultState
 
     [JsonIgnore]
     public bool IsComplete => Status.HasValue &&
-        (Status != MandatoryQualificationStatus.Passed || (Status == MandatoryQualificationStatus.Passed && EndDate.HasValue));
```

Comment:
They might be equivalent but I find the older version with the brackets easier to read.

Reply:
Strange - I don't recall actually changing that!

Reply:
Reverted it